courtlistener/process_cl.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import csv
+import logging
+import os
+import re
+import sys
+
+import pandas as pd
+
+from licensed_pile.licenses import PermissiveLicenses
+from licensed_pile.logs import configure_logging
+from licensed_pile.write import to_dolma
+
+SOURCE_NAME = "CourtListenerOpinion"
+
+csv.field_size_limit(sys.maxsize)
+
+logger = configure_logging("court-listener-opinion")
```

Comment:
prefer moving the configure_logging call under `if __name__ == "__main__":` and then have a `logger = logs.get_logger(...)` in the `process_court_listener` function.

Reply:
Will do.

Reply:
I think this is what you meant:
```python
def process_court_listener(file_path):

    logger = logging.getLogger("court-listener-opinion")
    ...
```
```python
if __name__ == "__main__":

    logger = configure_logging("court-listener-opinion")
    ...
```